### PR TITLE
Fixed small issues in docs.

### PR DESCRIPTION
**Pull request by Wise-Elm**
Fixed broken links, small formatting issue, and small error in cloning repository description. (This is my first contribution to anything. Please forgive any errors in how to properly contribute.)

### CODE REVIEW

File: docs/alerts.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ## Alerts
 
-Pass/fail based on metrics captured from the cluster is important in addition to checking the health status and recovery. Kraken supports alerting based on the queries defined by the user and modifies the return code of the run to determine pass/fail. It's especially useful in case of automated runs in CI where user won't be able to monitor the system. It uses [Kube-burner](https://kube-burner.readthedocs.io/en/latest/alerting.html) under the hood. This feature can be enabled in the [config](https://github.com/cloud-bulldozer/kraken/blob/master/config/config.yaml) by setting the following:
+Pass/fail based on metrics captured from the cluster is important in addition to checking the health status and recovery. Kraken supports alerting based on the queries defined by the user and modifies the return code of the run to determine pass/fail. It's especially useful in case of automated runs in CI where user won't be able to monitor the system. It uses [Kube-burner](https://kube-burner.readthedocs.io/en/latest/) under the hood. This feature can be enabled in the [config](https://github.com/cloud-bulldozer/kraken/blob/master/config/config.yaml) by setting the following:
```
Comment (chaitanyaenr) [inline docs/alerts.md:3]: s/cloud-bulldozer/chaos-kubox

Comment (Wise-Elm) [reply docs/alerts.md:3]: Let me know if I need to change anything.

**Comment by Wise-Elm** [comment]
> Thanks for the PR @Wise-Elm! Can we fix other references of cloud-bulldozer and kraken if possible please: https://github.com/chaos-kubox/krkn/search?p=1&q=cloud-bulldozer?

Sure thing. Based on your link I think I have a better understanding of what you need. I will work on that.

**Comment by Wise-Elm** [comment]
I did a quick read through your docs files and fixed some spelling and grammar errors where I found them, and tried to make files in a consistent format with each other. I fixed links in the docs as well as the main README.MD with the exception of the link on line 118, which I did not know how to resolve. I also did not know how to resolve the links in common_test_confit.yaml on lines 17 & 18. Hope this helps.

**Review by sanjacodes — APPROVED** [review]
This is a great and thorough PR - thank you for your contribution! We'll fix the rest as you mentioned in your [comment](https://github.com/chaos-kubox/krkn/pull/241#issuecomment-1116944130).

Fixes #239.

**Pull request merged**